### PR TITLE
Fix some links in README files

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Fix some links.

### Motivation

The repository is hardcoded in the `ddev` command, I created a PR to fix this too https://github.com/DataDog/integrations-core/pull/12930

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: fluxcd/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Need help? Contact [Datadog support][9].
 [4]: https://github.com/DataDog/integrations-extras/blob/master/fluxcd/datadog_checks/fluxcd/data/conf.yaml.example
 [5]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
 [6]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
-[7]: https://github.com/DataDog/integrations-core/blob/master/check/metadata.csv
-[8]: https://github.com/DataDog/integrations-core/blob/master/check/assets/service_checks.json
+[7]: https://github.com/DataDog/integrations-extras/blob/master/check/metadata.csv
```
Comment (jtappa) [inline fluxcd/README.md:63]:
```suggestion
[7]: https://github.com/DataDog/integrations-extras/blob/master/fluxcd/metadata.csv
```

File: fluxcd/README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Need help? Contact [Datadog support][9].
...
+[8]: https://github.com/DataDog/integrations-extras/blob/master/check/assets/service_checks.json
```
Comment (jtappa) [inline fluxcd/README.md:64]:
```suggestion
[8]: https://github.com/DataDog/integrations-extras/blob/master/fluxcd/assets/service_checks.json
```

File: pliant/README.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ Need help? Contact [Datadog support][7].
 
 [1]: https://pliant.io/
 [2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
-[3]: https://github.com/DataDog/integrations-core/blob/master/pliant/datadog_checks/pliant/data/conf.yaml.example
+[3]: https://github.com/DataDog/integrations-extras/blob/master/pliant/datadog_checks/pliant/data/conf.yaml.example
```
Comment (jtappa) [inline pliant/README.md:120]: This link 404's. I don't see any file like this in the pliant directory.

File: zscaler/README.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ Need help? Contact [Datadog support][7].
 
 [1]: https://www.zscaler.com/products/zscaler-internet-access
 [2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
-[3]: https://github.com/DataDog/integrations-core/blob/master/zscaler/datadog_checks/zscaler/data/conf.yaml.example
+[3]: https://github.com/DataDog/integrations-extras/blob/master/zscaler/datadog_checks/zscaler/data/conf.yaml.example
```
Comment (jtappa) [inline zscaler/README.md:134]: Same as previous: This link 404's. I don't see any file like this in the zscaler directory.

**Review by jtappa — CHANGES_REQUESTED** [review]

File: pliant/README.md (hunk 3 above)
Comment (FlorentClarret) [reply pliant/README.md:120]: 😓 sorry about that, I did not check if the link was actually used :/ Fixed.

File: zscaler/README.md (hunk 4 above)
Comment (FlorentClarret) [reply zscaler/README.md]: same, fixed! Thanks

**Review by jtappa — APPROVED** [review]

**Pull request merged**